mox/src/lib.rs

```diff
@@ -398,6 +402,25 @@ fn mangle_ident(ident: &mut syn::Ident) {
     }
 }
 
+fn dashes_to_underscores(
+    punctuated: Punctuated<proc_macro2::Ident, syn_rsx::punctuation::Dash>,
```

Comment:
If you add these types to the top-of-file imports this should read easier:

```suggestion
    punctuated: Punctuated<Ident, Dash>,
```

Reply:
Agreed. Not sure why I wrote it like that!